### ISSUE TITLE
Issue #2 [Priority-High]

### ISSUE THREAD

**Issue by srabbelier-google** [srabbelier-google/issue-export-test#2]
Original [issue 2](https://code.google.com/p/issue-export-test/issues/detail?id=2) created by chrsmith on 2015-01-07T17:57:59.000Z:

This is a new issue. Initially it has label &quot;Priority-Low&quot;.

**Comment by srabbelier-google** [comment]
Comment [#1](https://code.google.com/p/issue-export-test/issues/detail?id=2#c1) originally posted by chrsmith on 2015-01-07T17:58:10.000Z:

This is a comment, disregard.

**Comment by srabbelier-google** [comment]
Comment [#2](https://code.google.com/p/issue-export-test/issues/detail?id=2#c2) originally posted by chrsmith on 2015-01-07T17:58:26.000Z:

This comment sets the Priority label to High.

**Comment by srabbelier-google** [comment]
Comment [#3](https://code.google.com/p/issue-export-test/issues/detail?id=2#c3) originally posted by chrsmith on 2015-01-07T18:02:55.000Z:

&lt;empty&gt;

**Comment by srabbelier-google** [comment]
Comment [#4](https://code.google.com/p/issue-export-test/issues/detail?id=2#c4) originally posted by chrsmith on 2015-01-07T18:03:11.000Z:

This issue is now blocked on issue #&nbsp;7.
